python/src/spark_rapids_ml/umap.py

```diff
@@ -343,7 +343,7 @@ def __init__(self, sample_fraction: float = 1.0, **kwargs: Any) -> None:
         super().__init__()
         self.set_params(**kwargs)
         self.sample_fraction = sample_fraction
-        self.maxRecordsPerBatch = 10000
+        self.max_records_per_batch = 10000
```

Comment:
Can we set this in the same way as line 369, i.e. through "spark.sql.execution.arrow.maxRecordsPerBatch"?